src/index.html

```diff
@@ -15,8 +15,90 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          class="logo"
+          alt="Sitelogo"
```

Comment:
The `alt` attribute should describe the image content. It helps improve your page accessibility and SEO. For example, if the image is a logo of the company, a good `alt` attribute can be the company's name.